### PR TITLE
Improve the error message for an unknown opcode.

### PR DESCRIPTION
**Pull request by bentglasstube**
In the future this could also look at the opcode to see which extension
is needed.

### CODE REVIEW

**Comment by bentglasstube** [comment]
Forgot to mention this is related to issue #13

File: src/Machine.cpp [hunk 1]
```diff
@@ -115,7 +115,8 @@ void Machine::execute(uint16_t& opcode){
 
 	if(it != first_match.end()) (it->second)(opcode);
 	else {
-		std::cout << "No match found for " << std::hex << (int) opcode << "\n";
+		std::cout << "No match found for opcode " << std::hex << (int) opcode << "\n";
+		std::cout << "This could be because this ROM uses SCHIP or another extension which is not supported.\n"
```
Comment (LakshyAAAgrawal) [inline src/Machine.cpp:119]: "Which is not supported yet"

**Comment by LakshyAAAgrawal** [comment]
Thanks a lot for the PR @bentglasstube . I have left a comment. PLease fix that and I will merge the PR.

**Comment by LakshyAAAgrawal** [comment]
Thank you very much @bentglasstube !! Highly appreciate the PR.

**Pull request merged**